src/suggestionlistworker.cpp

```diff
@@ -26,7 +26,9 @@ void SuggestionListWorker::run()
         kiwix::SuggestionsList_t suggestions;
         reader->searchSuggestionsSmart(m_text.toStdString(), 15, suggestions);
         for (auto& suggestion: suggestions) {
-            url.setPath(QString::fromStdString(suggestion[1]));
+            QString path(QString::fromStdString(suggestion[1]));
+            path.push_front('/');
+            url.setPath(path);
```

Comment:
@asashnov Why exactly we need a leading `/`? I believe we should probably later deal with that, at the moment the URL has to be open. Do you know where is that?